src/App.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
+import Carousel from "./components/Carousel/Carousel";
+import UpcomingEventCard from "./components/UpcomingEventCard/UpcomingEventCard";
+import { FakeUpcomingEvents } from "./FakeUpcomingEvents";
```

Comment:
Volodymyr advised to avoid using semicolons, you can use VS code properties to fix it.